TODO

```diff
@@ -2470,7 +2470,6 @@ Features:
 * support crash reporting operation modes (https://live.gnome.org/GnomeOS/Design/Whiteboards/ProblemReporting)
 
 * tmpfiles:
-  - apply "x" on "D" too (see patch from William Douglas)
```

Comment:
how does this commit relate to #20641?

also, i guess you can add "Fixes: #1633" to this commit?